tests/behat/src/FixtureContext.php

```diff
@@ -164,8 +164,8 @@ public function iAttachTheFileToDropzone($path, $name)
         // Get path
         $filesPath = $this->getFilesPath();
         if ($filesPath) {
-            $fullPath = rtrim(realpath($filesPath), DIRECTORY_SEPARATOR).DIRECTORY_SEPARATOR.$path;
-            if (is_file($fullPath)) {
+            $fullPath = rtrim((string) realpath((string) $filesPath), DIRECTORY_SEPARATOR).DIRECTORY_SEPARATOR.$path;
```

Comment:
Is there any chance that `realpath` would not return a string here?

```suggestion
            $fullPath = rtrim(realpath((string) $filesPath), DIRECTORY_SEPARATOR).DIRECTORY_SEPARATOR.$path;
```

Reply:
`string|false`

https://www.php.net/manual/en/function.realpath.php

The code in the PR is automatically generated.  If a native function has a single return type there won't be a cast/ternary, if there's more than one type, such as in this instance, it will be cast/ternary